### PR TITLE
feat(js): upgrade Getting Started to v3

### PR DESCRIPTION
**Pull request by francoischalifour**
This updates the InstantSearch.js Getting Started to version 3, using the [beta version](https://cdn.jsdelivr.net/npm/instantsearch.js@beta) for now.

Demo: https://codesandbox.io/s/github/algolia/doc-code-samples/tree/csb-1545053595649/InstantSearch.js/getting-started

_(updated with CodeSandbox)_

### CODE REVIEW

File: InstantSearch.js/getting-started/package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "lint:fix": "npm run lint -- --fix"
   },
   "devDependencies": {
+    "@types/algoliasearch": "3.30.1",
```
Comment (Haroenv) [inline InstantSearch.js/getting-started/package.json]: is this really necessary? VS Code should pick those up automatically 😄

File: InstantSearch.js/getting-started/src/app.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
 /* global instantsearch */
+import algoliasearch from 'algoliasearch/lite';
```
Comment (Haroenv) [inline InstantSearch.js/getting-started/src/app.js]: if InstantSearch is global, this should probably be too?

File: InstantSearch.js/getting-started/package.json (hunk 1 above)
Comment (francoischalifour) [reply InstantSearch.js/getting-started/package.json]: That was added by CodeSandbox automatically. We can remove it.

File: InstantSearch.js/getting-started/src/app.js (hunk 2 above)
Comment (francoischalifour) [reply InstantSearch.js/getting-started/src/app.js]: Yep!

**Pull request merged**